components/bms_worker/include/BatteryMonitoring.h

```diff
@@ -74,9 +74,9 @@ typedef struct
     BMS_State_E state;
     bool        fault;
     uint16_t    balancing_cells;
-    BMS_Cell_S  cells[MAX_CELL_COUNT];      // [mv], precision 1mv
-    float32_t   pack_voltage;               // [mv], precision 1mv
-    float32_t   calculated_pack_voltage;    // [mv], precision 1mv
+    BMS_Cell_S  cells[MAX_CELL_COUNT];      // [V], precision 1V
```

Comment:
still not clear on where this 1v precision is coming from

Reply:
I've been told that when those values have been converted to float32_t, all the values were converted to the "default unit". Maybe I misunderstood though but this is what I've been told.